src/main/java/org/jenkinsci/pipeline_steps_doc_generator/HyperLocalPluginManager.java

```diff
@@ -1,7 +1,37 @@
 package org.jenkinsci.pipeline_steps_doc_generator;
 
+import static hudson.init.InitMilestone.PLUGINS_LISTED;
+
+import java.io.File;
```

Comment:
So was this your editor moving the java imports over the Jenkins imports?

Reply:
Yes, this was the default behavior of auto-organize the imports.